docs/linux/packaging.md

```diff
@@ -169,38 +169,81 @@ Building packages happen in the [Keyman source tree](https://github.com/keymanap
 
 The Keyman
 [`linux/scripts/jenkins.sh`](https://github.com/keymanapp/keyman/blob/master/linux/scripts/jenkins.sh)
-script can be used to create a source package (replace `packageName` with the name of the package,
-i.e. one of keyman, kmflcomp, libkmfl, and ibus-kmfl).
+script can be used to create a source package.
 
 ```bash
 cd linux
-./scripts/jenkins.sh ${packageName} ${DEBSIGNKEY}
+./scripts/jenkins.sh keyman ${DEBSIGNKEY}
 ```
 
-This creates a source package (`<packageName>_<version>-1.dsc`) and some `*.tar.?z` files in the source root directory for `keyman`.
+This creates a source package (`keyman_<version>-1.dsc`) and some `*.tar.?z`
+files in the source root directory for `keyman`.
 
 ci-builder-script's [`build-package`](https://github.com/sillsdev/ci-builder-scripts/blob/master/bash/build-package)
 script creates the binary packages:
 
 ```bash
-cd linux/${packageName}
+cd $KEYMAN_ROOT
```

Comment:
So we don't cd to `$KEYMAN_ROOT/linux` ?

Reply:
No, because the source package gets put in `$KEYMAN_ROOT`, so we need to be in that directory.